invoices_app/views.py

```diff
@@ -520,15 +521,9 @@ def _send_email_when_change_invoice_status(request, invoice):
 
     for user in users:
         activate(user.preferred_language)
-
-        subject = INVOICE_CHANGE_STATUS_TEXT_EMAIL.format(
-                invoice.invoice_number,
-                invoice.get_status_display(),
-            )
-        upper_text = INVOICE_CHANGE_STATUS_TEXT_EMAIL.format(
-                    invoice.invoice_number,
-                    invoice.get_status_display(),
-                )
+        texts = get_upper_and_email_text(invoice, request)
+        subject = texts[0]
+        upper_text = texts[1]
```

Comment:
instead of access to texts array, you could simply do
`subject, upper_text = get_upper_and_email_text(invoice, request)`

Reply:
Check this for more info https://en.wikipedia.org/wiki/Assignment_(computer_science)#Parallel_assignment